mappings/net/minecraft/entity/projectile/FishingBobberEntity.mapping

```diff
@@ -23,4 +30,12 @@ CLASS net/minecraft/class_1536 net/minecraft/entity/projectile/FishingBobberEnti
 		ARG 3 lureLevel
 		ARG 4 luckOfTheSeaLevel
 	METHOD method_6947 getOwner ()Lnet/minecraft/class_1657;
+	METHOD method_6949 tickFishingLogic (Lnet/minecraft/class_2338;)V
+		ARG 1 pos
 	METHOD method_6951 updateHookedEntityId ()V
+	METHOD method_6952 smoothenMovement ()V
+	METHOD method_6954 pullHookedEntity ()V
+	METHOD method_6957 use (Lnet/minecraft/class_1799;)I
+		ARG 1 usedItem
+	METHOD method_6958 checkForCollision ()V
+	METHOD method_6959 checkRemoval ()Z
```

Comment:
shouldBeRemoved?

Reply:
To me, `shouldBeRemoved` sounds like a pure function that just returns whether or not the bobber should be removed, however, calling this method when the conditions are fulfilled actually goes ahead and removes the bobber, too. The boolean returned is in fact whether or not it was removed, though.
<img width="557" alt="image" src="https://user-images.githubusercontent.com/827762/63226480-4370ad80-c1da-11e9-86a5-dbb543f47434.png">
I've attached a screenshot of the code with deobfuscated locals from when I was looking at it.

Reply:
`tryRemove`/`tryInvalidate`